servlet-map/src/main/resources/META-INF/resources/spring-map-jsp/index.jsp

```diff
@@ -121,6 +121,7 @@
             <%-- If logout url is present - so logout link --%>
             <c:when test="${!empty _logout_uri}">
                 <a href="${pageContext.request.contextPath}${_logout_uri}"><spring:message code="logout" text="Logout" /></a>
+                <a href="javascript:void(0);" id="oskari-profile-link"><spring:message code="account" text="Account" /></a>
```

Comment:
Also add a comment that personaldata is going to look for a link with that id so no-one will change it by mistake